packages/terra-clinical-item-view/tests/jest/__snapshots__/ItemView.test.jsx.snap

```diff
@@ -2,8 +2,6 @@ exports[`test should render 1 start theme display 1`] = `
 <div
   className="terraClinical-ItemView terraClinical-ItemView--oneColumn terraClinical-ItemView-accessory--alignCenter">
```

Comment:
I guess it's not necessary, but the intention was to leave it there since it's more a style setting on the item-view, that could potentially be moved in the code to sit on the accessory.